src/main/java/net/william278/velocitab/Velocitab.java

```diff
@@ -63,6 +64,7 @@ public class Velocitab {
     private static final int METRICS_ID = 18247;
     private Settings settings;
     private final ProxyServer server;
+    @Getter
     private final Logger logger;
```

Comment:
get rid of this `Getter` here -- no need to expose this; instead, let's just use the `log` method